### DISCUSSION TITLE
Town Population screen (with the well), "Buy Monsters:" window, allow the player to recruit only the same types of creatures that are already present in the hero's army

### DISCUSSION THREAD

**Discussion by LeHerosInconnu** [ihhub/fheroes2#10293] [General]
In this window, it would be nice to make that when a hero is present in the castle, a button is displayed that allows to recruit only the same types of creatures that are already present in the hero's army.
The creatures are directly added to the hero's army.

In fheroes2:

![Window change fh2 12](https://user-images.githubusercontent.com/43583994/97112669-efb6f980-16e5-11eb-9412-caf60e1512ff.png)

**Comment by ihhub** [comment]
This is a very corner case to solve and I don't think many players would appreciate usefulness of this feature.

I am moving this to the Discussion section.
